source/NVDAObjects/IAccessible/mscandui.py

```diff
@@ -213,10 +213,48 @@ def event_show(self):
 		item=MSCandUIWindow_candidateListItem(IAccessibleObject=self.IAccessibleObject,IAccessibleChildID=3)
 		reportSelectedCandidate(item)
 
+class ModernCandidateUICandidateItem(BaseCandidateItem):
+
+	def _get_parent(self):
+		# Candidate list in Microsoft Quick cannot be obtained in IAccessible _get_parent. Use _get_parent in NVDAObject.window
+		parent=super(IAccessible,self).parent
+		return parent
+
+	def _get_candidateCharacters(self):
+		return super(BaseCandidateItem,self).name
+
+	_candidateNumber=""
+
+	def _get_candidateNumber(self):
+		# The following property call sets candidateNumber

```

Comment:
In the current code, this is only the case the first time you're calling this. Is that expected and correct?